dp-dataset-exporter-xlsx.nomad

```diff
@@ -62,6 +62,9 @@ job "dp-dataset-exporter-xlsx" {
 
       vault {
         policies = ["dp-dataset-exporter-xlsx"]
+
+        change_mode   = "signal"
+        change_signal = "SIGUSR1"
```

Comment:
Should this be SIGTERM? As we wont the app to stop and then nomad should restart it with the new vault token